vuu-ui/sample-apps/feature-basket-trading/src/new-basket-panel/NewBasketPanel.css

```diff
@@ -8,16 +8,16 @@
 
 .vuuBasketNewBasketPanel-body {
     flex: 1 1 auto;
+    padding-top: 16px;
+    padding-bottom: 16px;
```

Comment:
```
    padding-top: 16px;
    padding-bottom: 16px
```
can be written in a single line as:
 `padding: 16px 0;`
I prefer the latter approach when we want the same padding for top/bottom or left/right to avoid repetition

Reply:
[7cb928c](https://github.com/ScottLogic/finos-vuu/pull/100/commits/7cb928c1009535383fe3037472c354e893617aab)

Reply:
me too